core/src/main/resources/hudson/model/Messages_fr.properties

```
@@ -113,6 +113,9 @@ Descriptor.From=(de <a href="{1}" rel="noopener noreferrer" target="_blank">{0}<
 Executor.NotAvailable=N/A
 
 FreeStyleProject.DisplayName=Construire un projet free-style
+FreeStyleProject.Description=\
+  Tâche classique qui récupère l''état d''au plus un Système de Gestion de Code, exécute les étapes de build en série, \
```

```suggestion
  Job legacy polyvalent qui récupère l''état depuis un outil de gestion de version au plus, exécute les étapes de build en série, \
```
Optional suggestion, more close to other existing translations of "job", "legacy" and "SCM"